portfolio/src/main/webapp/script.js

```diff
@@ -35,9 +35,32 @@ jQuery(document).ready(function($) {
   */
 
   async function fetchAndRenderQuote(){
-      const responseFromServer= await fetch('/hello');
+      const responseFromServer= await fetch('/quotes');
       const textFromResponse= await responseFromServer.json();
       const quote = textFromResponse[Math.floor(Math.random() * textFromResponse.length)];
       const quoteContainer = document.getElementById('quote-container');
       quoteContainer.innerText = quote;
   }
+
+  function requestTranslation() {
+    const text = document.getElementById('text').innerText;
+    const htmlsytle=document.getElementById('text').innerHTML;
```

Comment:
Typo:` htmlsytle`. Also run a format over this, I see equals signs without spacing here and there.